ocd-2679.sql

```diff
@@ -0,0 +1,65 @@
+-- Create new table to support relationship between user and certification_body table
+DROP TABLE IF EXISTS openchpl.user_certification_body_map;
+CREATE TABLE IF NOT EXISTS openchpl.user_certification_body_map (
+	id bigserial NOT NULL,
+	user_id bigint NOT NULL,
+	certification_body_id bigint NOT NULL,
+	retired bool NOT NULL DEFAULT false,
+	creation_date timestamp NOT NULL DEFAULT NOW(),
+	last_modified_date timestamp NOT NULL DEFAULT NOW(),
+	last_modified_user bigint NOT NULL,
+	deleted bool NOT NULL DEFAULT false,
+	CONSTRAINT user_certification_body_pk PRIMARY KEY (id),
+	CONSTRAINT user_fk FOREIGN KEY (user_id)
+		REFERENCES openchpl.user (user_id) 
+		MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT,
+	CONSTRAINT certification_body_fk FOREIGN KEY (certification_body_id)
+		REFERENCES openchpl.certification_body (certification_body_id) 
+		MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
+);
+
```

Comment:
Needs the audit triggers added here and in the audit file.

Reply:
updated in ocd-2679.sql and openchpl_audit.sql